Bottom: 25f9bb4a1d79c067a43925748ab5a7590ad041c8
Top:    c500b93f30fec0045315e1baa63eba4f4c5713fa
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:46:58 -0500

Fix exim4 logrotate configuration

The logrotate configuration was selecting every file, causing the rotated
copies to also be rotated, leading to an exponential growth of files.

This fix limits the rotation to the actual log files, excluding copies.


---
diff --git a/roles/docker-exim4/templates/dockercontainer.exim4.logrotate.j2 b/roles/docker-exim4/templates/dockercontainer.exim4.logrotate.j2
index ca03e8e8..9100576b 100644
--- a/roles/docker-exim4/templates/dockercontainer.exim4.logrotate.j2
+++ b/roles/docker-exim4/templates/dockercontainer.exim4.logrotate.j2
@@ -1,6 +1,6 @@
 # Docker container for exim4 logrotate script
 
-{{ exim4_syslog_datavolume_dir }}/exim4/* {
+{{ exim4_syslog_datavolume_dir }}/exim4/*log {
     daily
     dateext
     ifempty
